Collections/script.js

```diff
@@ -253,17 +256,161 @@ console.log(filterByAge(people, 30));
 
 // Max punktów do zdobycia: 7
 
+// Zadanie 10.
+
+
+/*
+function ageFilter(age, arr) {  // deklaruję funkcję z dwoma parametrami. Pierwszy przyjmować będzie wartość odpowiadającą za wiek, a drugi wskazuje tablicę na której będę wyszukiwał.
+    return arr.filter(value => value.age > age) // za pomocą metody filter zwracam elementy tablicy arr, których wartość dla klucza "age" jest większa niż zadeklarowany w funkcji argument "age"
+}
+
+console.log(ageFilter(30, people)) // drukuję zwrócone w funkcji wyfiltrowane elementy tablicy people
+*/
 
 // Zadanie 11.
 
+
+function aboutUsers(arr) {  // deklaruję funkcję, której jako parametr podaję tablicę danych
+
+    arr.forEach(user =>  // za pomocą metody forEach iteruję elementy tablicy i dla każdego elementu drukuję treść "wstrzykując" w nią za pomocą interpolacji stringa wybrane wartości pobrane za pomocą odniesienia się do właściwości tego elementu (user.name, user.age itd)
+        console.log(`The user's name is  ${user.name} and the user is ${user.age} years old. 
+        The user works as ${user.job} and drives ${user.car}.`))
+}
+
+aboutUsers(people) // wywołuję funkcję dla tablicy people
+
+
+console.log("-- druga część zadania 11 --") 
+
+
+function aboutUsersMore(arr) { // deklaruję funkcję, której jako parametr podaję tablicę danych
+arr.forEach(user =>  user.car.includes("BMW") || user.car.includes("Mustang") ? // tak jak w pierwszej części zadania stosuję metodę forEach jednak dodatkowo stosuję w pierwszej kolejności instrukcję warunkową ternary operator, której warunkiem jest obecność słów kluczowych BMW lub Mustang w wartości obiektu klucza car przeszukiwanej tablicy "arr".
+            console.log(`The user's name is  ${user.name} and the user is ${user.age} years old.  
+            The user works as ${user.job} and drives ${user.car} which is fun car.`) : console.log(`The user's name is  ${user.name} and the user is ${user.age} years old. 
+            The user works as ${user.job} and drives ${user.car} which is boring car.`)) // jeśli warunek jest true to drukuję treść jak w pierwszej częsci zadania dodając na koniec fragment "which is fun car" a w przypadku false (czyli pojazdów nie będących BMW lub Mustangiem) dodaję fragment "which is boring car"
+    }
+
+aboutUsersMore(people) // wywołuję funkcję dla tablicy people
+
+
+// -----> kolejne zadania zostawiłem zakomentowane każde osobno za pomocą /* */ aby nie wpływały na siebie 
+            
+
 // Zadanie 12.
+/*
+
+function adultMan(age, arr) { // deklaruj funkcję, której parametrami będą wiek oraz tablica danych
```

Comment:
W VSCode `ctrl + shift + p` -> "format document with" -> "JS and TS lang features"